### PR TITLE
[PDX-2019] Added speaker schedule, new sponsors

### PR DESCRIPTION
**Pull request by dev-dull**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

### CODE REVIEW

File: data/events/2019-portland.yml [hunk 1]
```diff
@@ -128,23 +137,222 @@ sponsor_levels:
...
-#    date: 2018-09-12
+  - title: "allie-richards"
+    date: 2019-09-11
+  - title: "matty-stratton"
+    date: 2019-09-11
+  - title: "ashina-sipiora"
+    date: 2019-09-11
+  - title: "aaron-aldrich-ignite"
+    date: 2019-09-11
+  - title: "nell-shamrell-harrington"
```
Comment (bridgetkromhout) [inline data/events/2019-portland.yml]: While it's a good idea to use the hyphen, this needs to be consistent across the rest of the PR if you want it to work.

Comment (dev-dull) [reply data/events/2019-portland.yml]: Do'h. Thanks, Bridget. Excellent catch. That's what I get for doing part of the work manually. PR has been updated.

File: content/events/2019-portland/program.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Title = "Program"
+Type = "program"
+Description = "Program for devopsdays portland 2019"
+Icons = "false"
++++
+
+<div class = "row">
+  <div class = "col">
+    <hr />
+    If Open Space is new to you, you may be interested in <a href="/pages/open-space-format">more details about Open Space</a>.
```
Comment (jyee) [inline content/events/2019-portland/program.md]: Should this be "Open Spaces" (plural)?

Comment (bridgetkromhout) [reply content/events/2019-portland/program.md]: Up to you; this is the language in our site-wide templates.

**Pull request merged**